_i18n/es/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,165 @@
+`/dirección/completa/de/monero-wallet-cli --generate-from-device <nuevo-nombre-de-monedero> --subaddress-lookahead 3:200`
 
-Where the full path is, intuitively, the actual path on your Mac OS X.
+Donde la dirección completa es, intuitivamente, la dirección actual en tu Mac OS X.
 
-7. The CLI will, after executing aforementioned command, prompt you for a password. Make sure to set a strong password and confirm it thereafter.
+6. La consola te preguntará después de ejecutar el comando anterior por una contraseña. Asegúrate de establecer una contraseña fuerte y de confirmarla después de ello.
 
-8. The Ledger will ask whether you want to export the private view key or not. First and foremost, your funds cannot be compromised with merely the private view key. Exporting the private view key enables the client (on the computer - Monero v0.12.2.0) to scan blocks looking for transactions that belong to your wallet / address. If this option is not utilized, the device (Ledger) will scan blocks, which will be significantly slower. There is, however, one caveat. That is, if your system gets compromised, the adversary will potentially be able to compromise your private view key as well, which is detrimental to privacy. This is virtually impossible when the private view key is not exported.
+7. Ledger preguntará si deseas exportar la clave privada o no. Primero y ante todo, tus fondos no pueden ser comprometidos con simplemente tu clave privada de visualización. Exportar la clave privada de visualización permite al cliente (en el ordenador - Monero v0.12.2.0) escanear bloques en busca de transacciones pertenecientes a tu monedero / dirección. Si esta opción no es utilizada, el dispositivo (Ledger) escaneará bloques, lo que será significativamente más lento. Sin embargo, hay una advertencia. Si tu sistema se ve comprometido, el adversario será potencialmente capaz de también comprometer tu clave privada, lo que es perjudicial para tu privacidad. Esto es virtualmente imposible cuando la clave privada no es exportada.
 
-9. You may have to hit confirm twice before it proceeds.
+8. Puedes tener que hacer clic en confirmar dos veces antes de proceder.
 
-10. Your Ledger Monero wallet will now be generated. Note that this may take up to 5-10 minutes. Furthermore, there will be no immediate feedback in the CLI nor on the Ledger.
+9. Tu monedero Ledger Monero será ahora generado. Ten en cuenta que esto puede tomar de 5 a 10 minutos. Además, no habrá realimentación inmediata en la consola de comandos o en la Ledger.
 
-11. `monero-wallet-cli` will start refreshing. Wait until it has fully refreshed.
+10. `monero-wallet-cli` comenzará a refrescarse. Espera a que se haya refrescado totalmente.
 
-12. Congratulations, you can now use your Ledger Monero wallet in conjunction with the CLI.
+Felicidades, ahora puedes utilizar tu monedero Ledger Monero en conjunto con la consola de comandos.
```

Comment:
I can tell by the context, but, who said that it was for monero-wallet-cli? We can just use it if it's about it, I understand that it can be just a short version of the full name (just CLI), but that acronym is for command-line interfase, and not for monero-wallet-cli. It can be used to have less characters but it's not correct.

If indeed CLI is for monero-wallet-cli, all those sentences should just use **monero-wallet-cli**.

Reply:
We can open up an issue about it. Could you open the issue about it?

Reply:
Yes I'll open it, it should be better.